bundles/org.openhab.binding.resol/README.md

```diff
@@ -0,0 +1,196 @@
+| adapterSerial        | String  | no       | Serialnumber of the device (informative only)              |
+
+
+## Thing Configuration
+
+Depending on the solar/heating controller you have attached to your VBUS there will be a "controller" and several other things like heat quantity meters, heating circuit controls, etc.
+These do not require any configuration parameters and will pop up in your inbox after the bridge has received data from them.
+The name of the devices is usually the Resol name with spaced replaced by _ and a "-Controller" suffix like "DeltaSol_MX-Controller".
+For configuration in files you can enable the logging with at least INFO level for the resol binding and search the logs for "ThingHandler for (.*) not registered." to identify the names of the things you can add for your VBUS devices.
+
+
+## Channels
+
+The channels of a thing are determined automatically based on the received VBUS data and are highly dependent on the used device.
+Here is a list of the channels of a DeltaSol MX with a heat quantity meter (HQM) and an extension module EM.
+The channels supported for your device can be seen after autodiscovery or 
+
+| channel                       | type                     | description                                        |
+|-------------------------------|--------------------------|----------------------------------------------------|
+| Pump_speed_relay_x            | Number:Dimensionless     | Percentage of the output state of relay 'x'        |
+| Temperature_sensor_x          | Number:Temperature       | Temperature sensor 'x' of the controller           |
+| Temperature_Module_y_Sensor_x | Number:Temperature       | Temperature sensor 'x' of the extension module 'y' |
+| Pressure_sensor_x             | Number:Pressure          | Pressure sensor 'x'                                |
+| Humidity_sensor_x             | Number:Dimensionless     | Humidity sensor 'x'                                |
+| Irradiation_sensor_x          | Number:Intensity         | Sunlight intensity sensor                          |
+| Output_M                      | Number:Dimensionless     | PWM/0-10V level value of the output 'M'            |
+| System_date                   | DateTime                 | Date and time of the controller clock              |
+| Error_mask                    | Number                   | Bitmask for the different errors                   |
+| Error_Sensor_line_broken-str  | String                   | Sensor line broken status (details for Error_mask) |
+| Flow_rate_sensor_x            | Number:VolumetricFlowRate| of sensor 'x'                                      |
+| Flow_set_temperature          | Number:Temperature       | Heating circuit set temperature                    |
+| Operating_state               | Number                   | Heating circuit operationg state                   |
+| Operating_state-str           | String                   | Heating circuit operationg state, as text          |
```

Comment:
No I didn't. I searched for such a concept (looking for enum) but didn't find the state options. - This seems to do the trick, while it's not obvious how localization should be handled. - The string channel comes with the localized text (in en, de, anf fr) already from the library. - And it could be quite some effort to change but probably worth the effort. I'll investigate - sadly tomorrow I'll be back in work :-)